scripts/setup_compute_stack.sh

```diff
@@ -26,7 +26,7 @@ MINIKUBE_RECREATE=${MINIKUBE_RECREATE:-true}
 START_MINIKUBE=${START_MINIKUBE:-true}
 CONFIGURE_K8S_COMPUTE=${CONFIGURE_K8S_COMPUTE:-true}
 COMPUTE_NAMESPACE=${COMPUTE_NAMESPACE:-nvm-disc}
-
+KEEPER_ARTIFACTS_FOLDER="${HOME}/nevermined-contracts/artifacts"
```

Comment:
This variable is already defined in the `start_nevermined.sh` file:
```
export KEEPER_ARTIFACTS_FOLDER="${NEVERMINED_HOME}/nevermined-contracts/artifacts"
```
So I don't think needs to be re-defined here